packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -151,6 +151,9 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
       if (e.ctrlKey || e.altKey || e.metaKey || e.shiftKey || e.pointerType === 'keyboard') {
         return;
       }
+      if (e.pointerType === 'virtual' && columnState.currentlyResizingColumn != null) {
+        stateRef.current.onColumnResizeEnd(item);
+      }
```

Comment:
Confirmed that double tapping to trigger a press on the visually hidden input via VO and Talkback now "confirm"/exit the resize operation, but there seems to be an issue in Android Chrome now where the focus is lost to the body upon triggering the press. VO is a bit better but it suffers from an issue where the press happens on the element under the visually hidden input which could be an entirely different column depending on how much resizing happened.

What if you have move focus to the column header in this case instead since the resize handles go away? I'm fine with handling this in a followup or after we confirm the overall behavior with accessibility if you'd like

Reply:
Focus should be moving to the header on column resize end, so I'm not sure what you're seeing. Maybe we can hop on a call tomorrow and discuss it more.